src/Misc/fileTools.ts

```diff
@@ -160,7 +160,7 @@ export class FileTools {
 
         if (typeof Image === "undefined" || (engine?._features.forceBitmapOverHTMLImageElement ?? false)) {
             FileTools.LoadFile(url, (data) => {
-                createImageBitmap(new Blob([data], { type: mimeType }), { premultiplyAlpha: "none" }).then((imgBmp) => {
+                engine?.createImageBitmap(new Blob([data], { type: mimeType }), { premultiplyAlpha: "none" }).then((imgBmp) => {

```

Comment:
You should probably do `engine!` instead, as else we will be stuck without any error message if engine is null/undefined (even if it's only theoritical).